### DISCUSSION TITLE
Is the latest docker image labeled "preview" production ready?

### DISCUSSION THREAD

**Discussion by igaldr** [getredash/redash#7224] [Q&A]
Hi
I see that there's a new docker image, labeled "preview"
see here: [https://hub.docker.com/r/redash/redash/tags](https://hub.docker.com/r/redash/redash/tags)

Is it safe to use?
I'm asking because the latest release is from 3 years ago
![image](https://github.com/user-attachments/assets/37b0b15f-c392-44b8-b965-eafcd6d6fcf4)

Thanks!

**Comment by phillipjohnson** [comment]
I would advise not using `redash/redash:preview` only because it's not tied to a specific version. But you can access the specific pre-release versions here: https://hub.docker.com/r/redash/preview/tags, e.g. `redash/preview:24.11.0-dev`.

Many of us are eagerly following this discussion awaiting a stable build: https://github.com/getredash/redash/discussions/6647. Some folks there are using the preview builds in production.
